### PR TITLE
Thivya/update yml

### PR DESCRIPTION
**Pull request by thivya-deriv**
This pr is to create github actions under e2e/deriv-app repo.
Card : https://app.clickup.com/t/20696747/BOT-1208
PR: https://github.com/deriv-com/e2e-deriv-app/pull/38

### CODE REVIEW

File: .github/workflows/cypress_cloud_manual.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        description: Select the environment
+        options:
+        - https://staging-app.deriv.com/
+        - https://app.deriv.com/
+      suite:
+        type: choice
+        description: Run smoke or full test suite
+        options:
+        - full
+        - wip/bot
```
Comment (markodonnell-deriv) [inline .github/workflows/cypress_cloud_manual.yml:17]: I've just just it should be dbot not bot :) And you will need to add the /sanity-tests folder to the path too

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Just a path change and also can you remove the extra lines in the 4 sanity tests files. Unless it was intended?

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**